app/views/qsets/_qset_list.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="row qset_list">
+  <% @qsets.each_with_index do |qset, index| %>
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        <h4 class="panel-title">
+          <div class="row valign">
+            <div class="col-sm-11">
```

Comment:
I'm not sure why we're using `.col-sm-11` here -- bootstrap default is 12 columns, but we aren't specifying the 12th column anywhere...?